glide.yaml

```diff
@@ -25,3 +25,5 @@ import:
 - package: github.com/jpillora/go-ogle-analytics
 - package: golang.org/x/net
 - package: golang.org/x/time
+- package: github.com/Azure/azure-sdk-for-go/storage
```

Comment:
It is no different to vendor the entire repo or specific dir under the repo.
Just 

```
package: github.com/Azure/azure-sdk-for-go
version ...
```

BTW, it's `v10.3.0-beta` now :)